docs/compilation/performance_tips.md

```diff
@@ -0,0 +1,505 @@
+# Performance tips
+
+The most costly operation in Concrete is the table lookup operation, so the primary goal of optimizing performance is to reduce the amount of table lookups and reduce the bit width of the inputs to table lookups.
+
+## Try different implementation strategies for complex operations
+
+Concrete provides multiple implementation strategies for these complex operations:
```

Comment:
Maybe adding that for this complex operations, Concrete doesn't yet select automatically the optimal one, and let the user tests?

Reply:
I think this line in the info box:
```
The default strategy is the one that doesn't increase the input bit width, even if it's less optimal than the others.
```
Tells that. But I can add it as a separate warning if you insist.